defaults/main.yml

```diff
@@ -22,5 +22,20 @@ gitlab_ldap_bind_dn: "CN=Username,CN=Users,DC=example,DC=com"
 gitlab_ldap_password: "password"
 gitlab_ldap_base: "DC=example,DC=com"
 
+# Timezone
+gitlab_time_zone: "CET"
+
+# Gitlab mail
+gitlab_email_from: 'gitlab@gitlab.com'
```

Comment:
This should probably be something more generic by default, like `example@example.com`; I don't like using real URLs for email addresses. Same goes for the reply to address default.